src/main/scala/com/github/unchama/seasonalevents/christmas/ChristmasItemData.scala

```diff
@@ -0,0 +1,323 @@
+      case 5 => 0.1
+      case _ => throw new IllegalArgumentException("不正なエンチャントレベルが指定されました。")
+    }
+    val isZombie = enemyType == EntityType.ZOMBIE || enemyType == EntityType.ZOMBIE_VILLAGER
+
+    (if (isZombie) 40 else 20) * rate
+  }
+
+  //endregion
+
+  //region THANATOSレプリカ
+
+  val christmasPickaxe: ItemStack = {
+    val displayName = Seq(
+      "T" -> RED,
+      "H" -> GOLD,
+      "A" -> YELLOW,
+      "N" -> GREEN,
+      "A" -> BLUE,
+      "T" -> DARK_AQUA,
+      "O" -> LIGHT_PURPLE,
+      "S" -> RED,
+      " Replica" -> WHITE
+    ).map { case (c, color) => s"$color$BOLD$ITALIC$c" }
+      .mkString
+    val enchants = Set(
+      (Enchantment.DIG_SPEED, 3),
+      (Enchantment.LUCK, 1)
+    )
+    val loreList = {
+      val enchDescription = enchants
+        .map { case (ench, lvl) => s"$RESET$GRAY${Util.getEnchantName(ench.getName, lvl)}" }
+        .toList
+      val lore = List(
+        s"$RESET${AQUA}耐久無限"
+      )
+
+      enchDescription ::: christmasItemBaseLore ::: lore
+    }.asJava
+
+    val itemMeta = Bukkit.getItemFactory.getItemMeta(Material.DIAMOND_PICKAXE).tap { meta =>
+      import meta._
+      setDisplayName(displayName)
+      setLore(loreList)
+      setUnbreakable(true)
+      addItemFlags(ItemFlag.HIDE_ENCHANTS)
+      enchants.foreach { case (ench, lvl) => addEnchant(ench, lvl, true) }
+    }
+
+    val pickaxe = new ItemStack(Material.DIAMOND_PICKAXE, 1)
+    pickaxe.setItemMeta(itemMeta)
+
+    new NBTItem(pickaxe).tap { nbtItem =>
+      import nbtItem._
+      setByte(NBTTagConstants.typeIdTag, 5.toByte)
+    }
+      .pipe(_.getItem)
+  }
+
+  //endregion
+
+  //region hristmasSock
+
+  val christmasSock: ItemStack = {
+    val loreList = List(
+      "Merry Christmas!",
+      s"クリスマスをお祝いして$RED${UNDERLINE}靴下${RESET}をどうぞ！",
+      s"$RED${UNDERLINE}アルカディア、エデン、ヴァルハラサーバー メインワールドの",
+      s"$RED${UNDERLINE}スポーン地点にいる村人に欲しい物を詰めてもらおう！"
+    ).map(str => s"$RESET$str")
+      .asJava
+
+    val itemMeta = Bukkit.getItemFactory.getItemMeta(Material.PAPER).tap { meta =>
+      import meta._
+      setDisplayName(s"${AQUA}靴下")
+      setLore(loreList)
+      addEnchant(Enchantment.DIG_SPEED, 1, true)
+      addItemFlags(ItemFlag.HIDE_ENCHANTS)
+    }
+
+    val itemStack = new ItemStack(Material.PAPER, 1)
+    itemStack.setItemMeta(itemMeta)
+
+    new NBTItem(itemStack).tap { item =>
+      import item._
+      setByte(NBTTagConstants.typeIdTag, 6.toByte)
+      setObject(NBTTagConstants.expiryDateTag, END_DATE)
+    }
+      .pipe(_.getItem)
+  }
+
+  //endregion
+
+  val christmasMebiusLore: List[String] = List(
+    "",
+    s"$RESET${WHITE}Merry Christmas! あなたに特別なMebiusを"
+  )
+
+  // SeichiAssistで呼ばれてるだけ
+  def christmasPlayerHead(head: SkullMeta): SkullMeta = {
+    val lore = List(
+      "",
+      s"$GREEN${ITALIC}大切なあなたへ。",
+      s"$YELLOW$UNDERLINE${ITALIC}Merry Christmas $EVENT_YEAR"
+    ).map(str => s"$RESET$str")
+      .asJava
+    head.setLore(lore)
+    head
```

Comment:
```suggestion
    head.tap(_.setLore(lore))
```
これ人に依るかつ上っ面な点なので無視してcloseでもokです